### ISSUE TITLE
ctypes-android 0.5.1 fails to compile

### ISSUE THREAD

**Issue by wagerlabs** [whitequark/opam-cross-android#19]
```
opam switch
...
4.02.3+32bit  C 4.02.3+32bit  OCaml 4.02.3 compiled in 32-bit mode for 64-bit Linux and OS X hosts
...
```

then 

```
opam install ctypes-android 
The following actions will be performed:
  ∗  install ctypes-android 0.5.1

=-=- Gathering sources =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 

=-=- Processing actions -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 

#=== ERROR while installing ctypes-android.0.5.1 ==============================#
These patches didn't apply at /Users/joelr/.opam/4.02.3+32bit/build/ctypes-android.0.5.1:
  - patches/cross-compilation.patch
```

Am I doing something wrong? Other packages compile just fine.

**Comment by whitequark** [comment]
No, this is certainly something I screwed up.

**Comment by whitequark** [comment]
Fixed in a5052e8921fe49f9e.

**Comment by wagerlabs** [comment]
Thanks!
